src/tools/rbd_wnbd/wnbd_handler.h

```diff
@@ -98,6 +98,7 @@ class WnbdHandler
     reply_tpool = new boost::asio::thread_pool(_io_reply_workers);
   }
 
+  int resize(uint64_t block_count);
```

Comment:
Nit: this is now in bytes:
```suggestion
  int resize(uint64_t new_size);
```